### DISCUSSION TITLE
I'm glad

### DISCUSSION THREAD

**Discussion by ehunt34** [3kh0/ext-remover#183] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I make one comment about potentially running other operating systems and you remove all of the scripts from here. I hope you know I still have your unenroll scripts and I plan to use them. Just because you have a power creep does not mean you can gate keep everyone and assume everyone here is a dumbass. Great maintainer @3kh0 , good job. I hope you're happy watching your project fall to pieces by administration and Google. I'm hereby going to use my exploit. Get fucked.

**Comment by ghost** [comment]
No, the exploit is still in readme. You're being a pissy toddler because I told you you can't do something reckless and immature. Frankly, if you act like THIS because you can't do something illegal, you shouldn't be here. Look in the mirror and think: what the hell am I doing? who do I think I am? The project isn't "falling to pieces", and it's not even his in the first place. I recommend some of you grow the fuck up because it's concerning how many of you want to do something this risky and still act like rampant toddlers. Sincerely: a dev

**Reply by ghost** [reply]
PS: Nice self react

**Reply by ehunt34** [reply]
If my school doesn't want me cracking their software maybe they shouldn't make two students commit suicide in one year.

**Reply by ghost** [reply]
That's not my problem, neither is it the software's.

**Reply by ehunt34** [reply]
Then fuck off and quit treating everyone like an idiot.

**Reply by ghost** [reply]
you are an idiot. Look at your post and think about how you're acting. Instead of reasonably asking questions, you get absolutely PISSED. I mean, if you're so smart that you can handle this, DO IT ON YOUR OWN. Quit acting like a tough revolution soldier and GROW UP

**Reply by ghost** [reply]
Ps: I won't "fuck off", I'm a dev for this project

**Reply by ehunt34** [reply]
For a developer you make very bad decisions with people trying to help your projects. I'm not replying further. Delete this. I don't care about your fucking project anymore.

**Reply by ghost** [reply]
Good for you, really proved it in this post. "bad decisions?" you mean I don't let 12 year olds do risky illegal things? veeery bad decisions indeed. I mean, you can just piss yourself somewhere else, or maybe you can GROW UP. But hey, be my guest. Keep acting like this, just another reason we shouldn't develop this project further. Thank you for your time

**Reply by Theblackmamba824** [reply]
please dont let people like this slow your motivation for this project down

**Reply by ghost** [reply]
@Theblackmamba824 Don't worry, this actually inspires me to go faster, so people don't do exactly this

**Reply by ghost** [reply]
People actually are this dumb... guy just wants to play Roblox all day not even caring about his whole life ahead of him

**Reply by Aka-but-nice** [reply]
Bro, can someone ban @ehunt34 i am not the only one who hates this dude

**Comment by bartholemuew** [comment]
bro has no rizz 💀

**Comment by 3kh0** [comment]
How did this discussion page get more toxic than the TN everything chat 💀

**Reply by ghost** [reply]
TN?

**Reply by 3kh0** [reply]
titanium network

**Reply by ghost** [reply]
Ok

**Reply by piplup7575** [reply]
lol!

**Comment by rwg100** [comment]
idk bro that sounds like a skill issue

**Reply by Aka-but-nice** [reply]
what happend to yeeyeeyt

**Reply by ghost** [reply]
@Aka-unblock I'm still here but now I'm developing some other things

**Reply by Aka-but-nice** [reply]
Alr,

**Comment by JelloFruitAlt** [comment]
![35262B2E-374D-4169-9D1F-4995BB2EC4C7](https://user-images.githubusercontent.com/104905957/210794326-b2fc22eb-ecc9-44b3-b2aa-202417ff1bb1.jpeg)
He thrives off negativity

**Reply by Aka-but-nice** [reply]
true this kid is retarded all he does it stare at a computer, he is the definition of a skid, he has no life, this kid try's to act smart by using smart words, but really he's five years old

**Reply by Arthura0121** [reply]
fr like just get a pc which is so much better than using a cheap Chromebook that barely works, like this child is acting so immature. @ehunt34 if u see this message, go give the phone back to ur mommy ok?

**Reply by Aka-but-nice** [reply]
brobro, got roasted

**Reply by ehunt34** [reply]
Can you like, shut the fuck up? If you think I'm doing this for myself, you're mistaken. Like I said, when your school causes two suicides in one year, drastic measures have to be taken. I seem to remember you tried to become a collaborator when you first joined just two minutes prior. You are doing the exact same thing as 3kh0, trying to lie about "making progress". At least he put the unenroll script back up.  I don't see why you get to have a say.

**Reply by Aka-but-nice** [reply]
@ehunt34 Can you like get a fucking life? Where is it huh? You have absolutely no life. All you do is complain, complain, and complain. Get a fucking life. Get out of here, if you're just going to act like a 5-year-old, go ahead. The only thing you can do is be a delinquent, grow up! Stop speaking like a baby, you always complain over and over again,  but each time you complain you're acting more childish. Just make sure to give your mom her phone back, it's gonna go dead.

**Reply by ehunt34** [reply]
If you had the time to be a collaborator you might as well reconsider your life choices. My life is locked in place. I know my future. This entire place is childish. yeeteeyt had a point when he said this entire place is full of children. I don't suspect you are an adult considering you're in this project anyway. All of your repos are just forks with nothing added to them anyway.

**Reply by Aka-but-nice** [reply]
Stfu, if you had known your so-called "future" why tf are you here? You are the real child in here, you are ruining everything just by being here. Stop acting grown and get a life, you are always being annoying as fuck. Why do you still come here, if each time you act like a baby, talking about "My life is locked in place" your hairline locked in place, stop looking up definitions of words to act smarter you are not smart, what you are is retarded, why are you still here if you actually, "Knew your future" you wouldn't be here. Gtfo

**Reply by Alex2Mayhem** [reply]
First of all your wailing out. just because you get no bitches means you can come in here threatening people. Your grades are prob like an F for failure at life. Second did you not just hear your goofy self saying "i have your unenrollment scripts" & "Watching your project fall apart by Administration and google "  Put those two fucking things together if its falling apart because of google and you have the scripts and plan to use them. How are then gonna work if google is tearing them to shreds bozo.

**Reply by Aka-but-nice** [reply]
Tell em'

**Comment by 3kh0** [comment]
Seems like someone is sad that no one knows about them 😢

**Reply by Aka-but-nice** [reply]
lmao

**Reply by rwg100** [reply]
TRADE OFFER:
i receive: everything you own on this project
you receive: nothing
